.github/workflows/lint-pr-titles.yaml

```diff
@@ -0,0 +1,25 @@
+name: PR Title Lint
+on:
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
+permissions:
```

Comment:
Our repo grants `read + write` permissions to `GITHUB_TOKEN` which seemed to be the issue with `GET_BUILD` because we granted write access . I've updated the permissions here so that within this workflow they specifically are read only. Originally, I wanted to leave a comment if the PR title failed to validate, but that would require write access and since it didn't seem necessary, I've left it out.